nav2_collision_monitor/launch/collision_monitor_node.launch.py

```diff
@@ -55,43 +60,69 @@ def generate_launch_description():
         default_value=os.path.join(package_dir, 'params', 'collision_monitor_params.yaml'),
         description='Full path to the ROS2 parameters file to use for all launched nodes')
 
-    # Create our own temporary YAML files that include substitutions
-    param_substitutions = {
-        'use_sim_time': use_sim_time}
-
-    configured_params = RewrittenYaml(
```

Comment:
I'm pretty sure this is important (?)

Reply:
It was initially removed, since we have no more substitution rewrites in parameters. However, you are right: I've thought twice and checked that `RewrittenYaml` in addtion to parameter rewrites is also making namespace substitution, which was out of support for this script.

Added support the non-root namespace substitution in the https://github.com/ros-planning/navigation2/pull/3604/commits/65c762168955107ed7ad209b8ba6172884379c9c commit.